### PR TITLE
show workflow config error on file view also

### PR DESCRIPTION
**Pull request by a1012112796**
As title, follow #24124

image

### CODE REVIEW

File: modules/actions/workflows.go [hunk 1]
```diff
@@ -29,6 +29,18 @@ func init() {
...
+func IsWorkFlow(path string) bool {
+	if (!strings.HasSuffix(path, ".yaml")) && (!strings.HasSuffix(path, ".yml")) {
+		return false
+	}
+
+	if strings.HasPrefix(path, ".gitea/workflows") {
+		return true
+	}
+
+	return strings.HasPrefix(path, ".github/workflows")
```
Comment (wxiaoguang) [inline modules/actions/workflows.go]:
```suggestion
	return strings.HasPrefix(path, ".gitea/workflows") || strings.HasPrefix(path, ".github/workflows")
```

File: routers/web/repo/view.go [hunk 2]
```diff
@@ -348,6 +350,16 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		if issueConfigErr != nil {
 			ctx.Data["FileError"] = strings.TrimSpace(issueConfigErr.Error())
 		}
+	} else if actions.IsWorkFlow(ctx.Repo.TreePath) {
+		content, err := actions.GetContentFromEntry(entry)
+		if err != nil {
+			ctx.Error(http.StatusInternalServerError, err.Error())
+			return
+		}
```
Comment (wxiaoguang) [inline routers/web/repo/view.go]: I guess we can just ignore (log.Error) this error, there are too many 500 errors in Gitea, most of them could be avoided.

File: modules/actions/workflows.go [hunk 3]
```diff
@@ -29,6 +29,14 @@ func init() {
 	}
 }
 
+func IsWorkFlow(path string) bool {
```
Comment (wxiaoguang) [inline modules/actions/workflows.go]: This function name seems inconsistent. Other function names are `ListWorkflows` `DetectWorkflows` : Work[f]low

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by techknowlogick** [comment]
🤖 🎺

**Pull request merged**